enabler/index.html

```diff
@@ -169,7 +169,8 @@ <h1 class="h3 mb-3 font-weight-normal">Dev Site</h1>
                 });
 
                 if (builder.componentHasSubmit) {
-                  if (selectedValue === 'card') {
+                  const methodsRequireMounting = ['eps', 'card', 'klarna', 'klarna_paynow', 'klarna_account'];
+                  if (methodsRequireMounting.includes(selectedValue)) {
```

Comment:
I am a bit unsure how the dev-tools should work with regards to the:

1. custom pay button
2. componentHasSubmit property
3. mounting logic

Without this change these payment methods won't work with the current dev-tools but it sometimes renders the pay button even if it's not needed.

Reply:
I believe we need to improve upon the dev-tools in a further ticket/work and outside of the scope of this ticket.